worlds/witness/hints.py

```diff
@@ -187,15 +187,25 @@ def get_always_hint_items(world: "WitnessWorld") -> List[str]:
     return always
 
 
-def get_always_hint_locations(_: "WitnessWorld") -> List[str]:
-    return [
+def get_always_hint_locations(world: "WitnessWorld") -> List[str]:
+    always = [
         "Challenge Vault Box",
         "Mountain Bottom Floor Discard",
         "Theater Eclipse EP",
         "Shipwreck Couch EP",
         "Mountainside Cloud Cycle EP",
     ]
 
+    # Add Obelisk Sides that contain EPs that are meant to be hinted, if they are necessary to complete the Obelisk Side
+    if world.options.EP_difficulty == "eclipse":
+        always.append("Town Obelisk Side 6")  # Eclipse EP
+
+    if world.options.EP_difficulty > 1:
```

Comment:
```suggestion
    if world.options.EP_difficulty != "normal":
```
I'm pretty sure this is what was intended, since `> 1` is only Eclipse